docs/metric/README.md

```diff
@@ -0,0 +1,47 @@
+
+## Minio Prometheus Metric
+
+Minio server exposes an endpoint for Promethueus to scrape server data at `/minio/metric`.
+
+### Prometheus  probe
```

Comment:
extra space after Prometheus

Reply:
Deleted the extra space after Prometheus